src/platform/user/profile/utilities/index.js

```diff
@@ -12,7 +12,7 @@ import {
 const commonServices = {
   EMIS: 'EMIS',
```

Comment:
☝️ TODO: This line can be removed AFTER https://github.com/department-of-veterans-affairs/vets-api/pull/15393 is merged.